R/helpers.R

```diff
@@ -161,3 +161,17 @@ log_tictoc <- function(..., level = INFO, namespace = NA_character_) {
 
 }
 tictocs <- new.env()
+
+
+#' Logs the error message to console before failing
+#' @param expression call
+#' @export
+#' @examples \dontrun{
+#' log_failure('foobar')
+#' log_failure(foobar)
+#' }
+log_failure <- function(expression) {
+  tryCatch(expression, error = function(e) {
+    log_error(e$message)
+  })
```

Comment:
We still need to throw the captured exception after logging I think:

```suggestion
  e
  })
```

Reply:
actually, we need `stop(e)` here